### PR TITLE
Translate "Ruby 3.3.0-preview2 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818, ##3113

Actual diff is  17fdce371c212969ea22553ef4e0a605929e3aac
with small fix on preview1 title ;) e4727d4065a851056277d1954e2d3b46496bb5fd

### CODE REVIEW

**Comment by riseshia** [comment]
could you review this? @ruby/www-ruby-lang-org-i18n-ko

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+* Ruby용 JIT 개발에 관심이 있다면 [RubyKaigi 2023 3일차에 있었던 k0kubun의 발표](https://rubykaigi.org/2023/presentations/k0kubun.html#day3)를 확인하세요.
+
+## Bison을 Lrama로 대체
+
+* Bison 대신 [Lrama LALR 파서 생성기](https://github.com/yui-knk/lrama)를 사용하도록 변경했습니다. [Feature #19637](https://bugs.ruby-lang.org/issues/19637)
+  * 관심이 있다면 [Ruby 파서의 미래 비전](https://rubykaigi.org/2023/presentations/spikeolaf.html)을 참고하세요.
+
+## YJIT
+
+* 3.2 버전 대비 성능 대폭 향상
```
Comment (yous) [inline ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md:30]: I think this is a summary of improvements. The preview1 article said "Significant performance improvements," and I think that can also be updated.

```suggestion
* 3.2 버전 대비 주요 성능 개선 사항
```

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  * 선택적 인수가 포함된 더 많은 호출이 컴파일됩니다.
+  * 예외 처리기도 컴파일됩니다.
+  * 클래스의 인스턴스 변수가 가지는 객체 형상의 조합이 매우 복잡한 경우에도
+    컴파일되기 전의 바이트 코드로 전환되지 않습니다.
+  * 지원되지 않는 호출 타입은 이제 컴파일되기 전의 바이트 코드로 전환되지 않습니다.
+  * `Integer#!=`, `String#!=`, `Kernel#block_given?`, `Kernel#is_a?`,
+    `Kernel#instance_of?`, `Module#===`은 특별히 최적화되었습니다.
+  * 이제 optcarrot에서 인터프리터보다 3배 이상 빠릅니다!
+* 컴파일된 코드의 메타데이터가 훨씬 적은 메모리를 사용합니다.
+* ARM64에서의 코드 생성 개선
```
Comment (yous) [inline ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md:42]:
```suggestion
* ARM64에서 더 작은 코드 생성
```

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+* 일시 중지 모드에서 YJIT을 시작한 다음 나중에 수동으로 재개하는 옵션 추가
+  * `--yjit-pause`와 `RubyVM::YJIT.resume`
+  * 애플리케이션 부팅이 완료된 후에만 YJIT을 재개할 수 있습니다.
+* `--yjit-stats`로부터 생성된 `ratio_in_yjit` 통계는 릴리스 빌드에서도 이용가능하며,
```
Comment (yous) [inline ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md:46]:
```suggestion
* `--yjit-stats`로부터 생성된 `ratio_in_yjit` 통계는 릴리스 빌드에서도 이용 가능하며,
```

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+## 성능 향상
+
+* `defined?(@ivar)`가 객체 형상에 맞게 최적화되었습니다.
+
+## 그 이외의 3.2 이후로 주목할 만한 변경
+
+### IRB
+
+IRB에 여러 기능 강화가 있습니다. 다음과 같은 내용이 포함됩니다.
```
Comment (yous) [inline ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md:68]:
```suggestion
IRB에 여러 개선 사항이 추가됩니다. 다음과 같은 내용이 포함됩니다.
```

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+- 고급 `irb:rdbg` 통합 기능은 `pry-byebug`와 동등한 디버깅 경험을 제공합니다. ([doc](https://github.com/ruby/irb#debugging-with-irb)).
```
Comment (yous) [inline ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md:70]:
```suggestion
- 고급 `irb:rdbg` 통합 기능은 `pry-byebug`와 동등한 디버깅 경험을 제공합니다. ([문서](https://github.com/ruby/irb#debugging-with-irb)).
```

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+- `ls`와 `show_cmds`와 같은 명령어에서 페이징을 지원합니다.
+- `ls`와 `show_source` 명령어에서 더 정확하고 유용한 정보를 제공합니다.
+
+또한, IRB는 앞으로의 개선을 더 쉽게 하기 위해 방대한 리팩토링과 수십 개의 버그를 수정했습니다.
```
Comment (yous) [inline ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md:74]:
```suggestion
또한, IRB는 앞으로의 개선을 더 쉽게 하기 위해 방대한 리팩토링을 진행했고 수십 개의 버그를 수정했습니다.
```

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: fixed 3450631a3ec3cf1879bfdb2494cb821c05b6f71a

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: fixed 3450631a3ec3cf1879bfdb2494cb821c05b6f71a

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md (hunk 4 above)
Comment (riseshia) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: fixed 3450631a3ec3cf1879bfdb2494cb821c05b6f71a

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md (hunk 5 above)
Comment (riseshia) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: fixed 3450631a3ec3cf1879bfdb2494cb821c05b6f71a

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md (hunk 6 above)
Comment (riseshia) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: fixed 3450631a3ec3cf1879bfdb2494cb821c05b6f71a

File: ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md (hunk 1 above)
Comment (yous) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: Can you update preview1 article too?

Comment (riseshia) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: sure~  ebbb5224d

Comment (yous) [reply ko/news/_posts/2023-09-14-ruby-3-3-0-preview2-released.md]: Thanks!

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks for review!! :bow:

**Pull request merged**